doc/playbook.rst

```diff
@@ -79,48 +88,65 @@ From the ``main`` branch you can release a new major version (``X.0.0``), a new
+   **The following changes will trigger a major release:**
 
-If you are planning to update the Science Pipelines tag, you should also check the `Science Pipelines changelog <https://lsst-dm.github.io/lsst_git_changelog/weekly/>`_.
-In practice, almost any Science Pipelines update is at least a minor version, because new features are added constantly.
-In the future, there may be "patched weekly" builds, which would justify a patch version of Prompt Processing.
+   For the ``prompt_processing`` service:
 
-For the ``prompt_processing`` service, a new major version is triggered by any of the following:
+   * Incompatibility with old fanned-out ``nextVisit`` messages (almost any change to ``Visit`` qualifies)
+   * Incompatibility with an old `APDB schema`_, `ApdbSql`_, `ApdbCassandra`_, or `ApdbCassandraReplica`_ version (see `DMTN-269`_ for the distinction)
+   * Incompatibility with an old `Butler dimensions-config`_ version
+   * A new major version of the `Alerts schema`_ (see `DMTN-093`_ for details)
 
-* Incompatibility with old fanned-out ``nextVisit`` messages (almost any change to ``Visit`` qualifies)
-* Incompatibility with an old `APDB schema`_, `ApdbSql`_, `ApdbCassandra`_, or `ApdbCassandraReplica`_ version (see `DMTN-269`_ for the distinction)
-* Incompatibility with an old `Butler dimensions-config`_ version
-* A new major version of the `Alerts schema`_ (see `DMTN-093`_ for details)
+   For the `next_visit_fan_out`_ service:
 
-For the `next_visit_fan_out`_ service, a new major version is triggered by any of the following:
+   * Incompatibility with old Summit ``nextVisit`` messages
+   * Breaking changes in the fanned-out ``nextVisit`` messages (almost any change to ``NextVisitModel`` qualifies)
 
-* Incompatibility with old Summit ``nextVisit`` messages
-* Breaking changes in the fanned-out ``nextVisit`` messages (almost any change to ``NextVisitModel`` qualifies)
+#. Create a Release
 
-2. Create a Release
+   On GitHub.com, navigate to the main page of the repository.
+   To the right of the list of files, click **Releases**.
+   At the top of the page, click **Draft a new release**.
+   Type a tag using semantic versioning described in the previous section.
+   The Target should be the main branch.
 
-On GitHub.com, navigate to the main page of the repository.
-To the right of the list of files, click **Releases**.
-At the top of the page, click **Draft a new release**.
-Type a tag using semantic versioning described in the previous section.
-The Target should be the main branch.
+   .. note::
 
-Select **Generate Release Notes**.
-This will generate a list of commit summaries and of submitters.
-Add text as follows.
+      The `auto` option is unreliable if there are no changes in Prompt Processing itself.
+      In these cases, if there weren't any eligible Pull Requests since the last tag, it will default to choosing the version before the most recent Pull Request.
```

Comment:
Maybe "In these cases, it may choose an older version than the latest" is enough?